pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1840,7 +1840,7 @@ cdef class YearOffset(SingleConstructorOffset):
     """
     _attributes = tuple(["n", "normalize", "month"])
 
-    # _default_month: int  # FIXME: python annotation here breaks things
+    # FIXME: python annotation here breaks things
```

Comment:
do we know what thi refers?

Reply:
putting a python-style annotation `_default_month: int` here led to (i think) compile-time errors

Reply:
Yes, that's right. I rolled back this change for now, as this part doesn't concern quarters. If OK, I hope to refactor/combine the  QuarterOffset and YearOffset and deal with it at some point.